src/server/game/Movement/MovementPacketSender.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2008-2017 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MOVEMENT_SENDER_H
+#define MOVEMENT_SENDER_H
+
+#include "Unit.h"
```

Comment:
This file does not need to include any of these.
Move `MAX_MOVE_TYPE` and `enum UnitMoveType` to UnitDefines.h and include that

Pass `MovementInfo` by const ref and forward declare everything